homeassistant/components/verisure/lock.py

```diff
@@ -98,6 +98,11 @@ def available(self) -> bool:
     def changed_by(self) -> str | None:
         """Last change triggered by."""
         return self.coordinator.data["locks"][self.serial_number].get("userString")
+    
+    @property
+    def changed_method(self) -> str | None:
```

Comment:
Nothing seems to use this property.

Reply:
Thank you for your review. This is the first time I look at the HA code, som my inexperience is exposed. I thought the properties would automatically be exposed as attributes to the lock entity. Then the "method" attribute could be used in automations.

Any pointers on how to expose "change_method" as an attributes to the lock entity?

Reply:
Extra state attributes can be set by implementing the property `extra_state_attributes`. A codeowner should approve the addition of new state attributes.

Reply:
I think it's ok to add it as a state attribute 👍

Reply:
Thanks both, updated the code with the extra state attribute, please review